tests/commands/test-kob-upgrade.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+function __test_kob_init
+{
+    if [[ ! -d $KOBMAN_DIR ]]; then
+        echo "kobman not found"
+        echo "Please install kobman first and try again"
+        exit 1
+    else
+        source $KOBMAN_DIR/bin/kobman-init.sh
+        source $KOBMAN_DIR/src/kobman-utils.sh
+        __kobman_echo_no_colour "kob found"
+    fi
+
+    if [[ -z $FAKE_REPO_NAMESPACE ]]; then
+        __kobman_echo_no_colour "No dummy namespace found"
+        __kobman_echo_no_colour "Cannot run test script without fake repository"
+        __kobman_echo_no_colour "Exiting!!!"
+        exit 1
+    fi
+}
+
+function __test_kob_execute
+{
+    __kobman_echo_no_colour "Executing commands"
+    kob -V >> $HOME/old.txt
```

Comment:
@asa1997   following a naming convention for the output file. eg. ts1_result.out.,ts2_result.out will help here
so you will have ts{1..n}_result.out files to validate in the validate section.

In the cleanup you remove all *.out or output files